os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/flash_api.c

```diff
@@ -384,6 +384,16 @@ int  flash_stream_write(flash_t *obj, u32 address, u32 len, u8 * data)
 		}
 
 		addr_begin = (((addr_begin-1) >> 2) + 1) << 2;
+		for(;size >= 256 ;size -= 256){
+			_memcpy(write_data, buffer, 256);
+			FLASH_TxData256B(addr_begin, 256, write_data);
+#ifdef MICRON_N25Q00AA
+			FLASH_ReadFlagStatusReg();
+#endif
+			buffer += 256;

```

Comment:
why do we use only 256 bytes here? can we use macro and replace these hardcoding

Reply:
The 256 byte used here is only to be consistent with **FLASH_TxData256B**.